integreat_cms/cms/models/events/event.py

```diff
@@ -35,9 +35,9 @@ def filter_upcoming(self, from_date=None):
         :return: The Queryset of events after the given date
         :rtype: ~integreat_cms.cms.models.events.event.EventQuerySet
         """
-        from_date = from_date or date.today()
+        from_date = from_date or timezone.now().date()
```

Comment:
`from django.utils import timezone` is missing at the top of the file

Reply:
Done